lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -53,6 +68,9 @@ function getHTMLImages(allElements) {
       naturalHeight: element.naturalHeight,
       attributeWidth: element.getAttribute('width') || '',
       attributeHeight: element.getAttribute('height') || '',
+      cssWidth: '', // this will get overwritten below
```

Comment:
if these are going to be `string | undefined` (or optional `string`), does it make more sense to initialize to `undefined` instead of `''`? Then it may be clearer that the value at the end of all this will be either a meaningful string or `undefined`

Reply:
Yeah, I like the idea of having it be optional `string`, so I'll make sure to change those initializations to undefined!